### PR TITLE
[client](rf) remove the fail over mechanism

### PR DESCRIPTION
**Pull request by ThibaultRiviere**

### CODE REVIEW

**Comment by ThibaultRiviere** [comment]
@ironman-machine can i do that ?

**Comment by ThibaultRiviere** [comment]
@ironman-machine with the good branches please
SCALITY_INTEGRATION_BRANCH=fix/nimrod-build-dep
SCALITY_FEDERATION_BRANCH=ft/dep-patcher-ansible

**Comment by ThibaultRiviere** [comment]
Fix #80

**Comment by ThibaultRiviere** [comment]
End to end passed !

**Comment by MathieuCassagne** [comment]
LGTM :+1:

File: lib/RESTClient.js [hunk 1]
```diff
@@ -336,6 +289,7 @@ class RESTClient {
      * @params {function} callback - callback
      */
     request(method, beginPath, log, params, data, callback) {
+        log.debug('sending request', { httpMethod: method, beginPath });
         let path = beginPath;
         assert(typeof callback === 'function', 'callback must be a function');
         const headers = {
```
Comment (ttvi) [inline lib/RESTClient.js]: Not related to this PR but shouldn't `content-length` be equal to `data.length` in the case `data` is available?

**Comment by AntoninCoulibaly** [comment]
can you put the Fix #80 in the commit message ? 
:+1:

**Comment by ThibaultRiviere** [comment]
Addressed @AntoninCoulibaly

**Comment by ThibaultRiviere** [comment]
@ironman-machine with the good branches please
SCALITY_INTEGRATION_BRANCH=fix/nimrod-build-dep
SCALITY_FEDERATION_BRANCH=ft/dep-patcher-ansible

**Comment by ThibaultRiviere** [comment]
Rebase on master after the last merge !

**Comment by alexandre-merle** [comment]
@ironman-machine did you work ?

**Comment by ThibaultRiviere** [comment]
@ironman-machine try

**Comment by ThibaultRiviere** [comment]
@ironman-machine clean try

**Comment by ThibaultRiviere** [comment]
@RachedBenMustapha ok for merging ?

**Comment by ThibaultRiviere** [comment]
@ironman-machine try

**Comment by DavidPineauScality** [comment]
@vrancurel @RachedBenMustapha Is this still something we want ? We'd need to merge it if it is.

**Comment by DavidPineauScality** [comment]
@ironman-machine r+

**Comment by DavidPineauScality** [comment]
@ironman-machine r- clean r+

**Pull request merged**